codespell_lib/data/dictionary.txt

```diff
@@ -7209,7 +7211,7 @@ counterintuive->counter intuitive
 counterpar->counterpart
 counterpoart->counterpart
 countie's->counties, counties', county's,
-countinueq->continueq
+countinueq->continueq, continue,
```

Comment:
Can you add `countinue->continue` too

Reply:
done

Reply:
Should `continueq->continue` be added to the code dictionary? :shrug:

Reply:
> Should `continueq->continue` be added to the code dictionary? shrug

Possibly, someone needs to look on grep.app and see if it's ever a typo for continue. Apart from hardcore kernel devs using their muscle-memory it seems unlikely to me, but I'm happy to be corrected.

Reply:
It was introduced in #1327 
if one searches the exact word, only codespell shows up now.
If one searches the the kernel repo for all hits of the word,  some variable names show up.

[1] https://grep.app/search?q=continueq&words=true 
[2] https://grep.app/search?q=continueq&filter[repo][0]=torvalds/linux

Reply:
I did a checkout of the linux kernel repo and investigated a little bit.

The typo and the suggestion both occur in "drivers/scsi/lpfc/lpfc_sli.{h,c}" and don't (really) change since 2011.

Reply:
> It was introduced in #1327
> if one searches the exact word, only codespell shows up now.
> If one searches the the kernel repo for all hits of the word, some variable names show up.

So it would catch and fix snake_case typos then.

Reply:
Yes. It would (if codespell would be able to catch snake_case) find two times a kernel driver.

How to proceed? It seems the last point of discussion in this MR.
For me it is fine to be commited as it is.

Reply:
> Yes. It would (if codespell would be able to catch snake_case) find two times a kernel driver.

It can already if you change the regex.

> How to proceed? It seems the last point of discussion in this MR.
> For me it is fine to be commited as it is.

Yeah what you've got is fine as is. If we can find proof of people typing continueq when they mean continue we could add that to the code dictionary.